frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/cater-capacity/cater-capacity.component.html

```diff
@@ -0,0 +1,29 @@
+<div *ngIf="canCaterFor" style="max-width: 20vw">
+    <span>{{
+        type === 'material' ? 'Dieses Material ' : 'Diese Einsatzkraft '
+    }}</span>
+
+    <span *ngIf="(canCaterFor | caterCapacityCount) === 0; else caterInfo"
+        >kann keinen Patienten helfen.</span
+    >
+    <ng-template #caterInfo>
+        <span>{{
+            canCaterFor.logicalOperator === 'or'
+                ? 'kann nur Patienten einer Verletzungsschwere gleichzeitig helfen.'
+                : 'kann Patienten verschiedener Verletzungsschweren helfen und dabei statt schwerer auch leichter verletzten helfen.'
+        }}</span>
+
+        <div class="container">
+            <div *ngFor="let status of caterForStatuses" class="row gx-2">
+                <div class="col-6 text-end">
+                    <app-patient-status-badge
+                        [status]="status"
+                    ></app-patient-status-badge>
+                </div>
+                <div class="col-6">
+                    <span class="">{{ canCaterFor[status] }}</span>
```

Comment:
Dust

```suggestion
                    {{ canCaterFor[status] }}
```

Text doesn't always need to be in a `<span></span>`.

Reply:
Or you give it a different style:
```suggestion
                    <span class="font-monospace">{{ canCaterFor[status] }}</span>
```